openrtb_ext/request_wrapper.go

```diff
@@ -95,6 +95,12 @@ func (rw *RequestWrapper) GetImp() []*ImpWrapper {
 
 func (rw *RequestWrapper) SetImp(imps []*ImpWrapper) {
 	rw.impWrappers = imps
+	imparr := make([]openrtb2.Imp, len(imps), len(imps))
```

Comment:
Nitpick: There is not need to specify both a length and capacity when they are the same. Just a single term will suffice here.

Reply:
Thanks